hw/Makefile

```diff
@@ -2,6 +2,10 @@ CUR_DIR := $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
 PRJ_DIR := $(realpath ${CUR_DIR}/../)
 export PRJ_DIR
 
+REG_OUTPUT_DIR    ?= ${PRJ_DIR}/build/regs-generated
```

Comment:
@eunchan towoe 
I refactored this to generate the DV and SW regs at $REPO_TOP/build/regs-generated.
Please let me know if you see a problem with this.